### DISCUSSION TITLE
a class for creating LogFile?

### DISCUSSION THREAD

**Discussion by legotronic** [pybricks/support#706] [Ideas]
We are test-driving the SPIKE hardware and expressing our effort to make live values more "visible". Has a class with LogFile functions also been thought of?

This would make our "engineering work" much easier - we are just used to the Display@EV3... 😊

**Comment by laurensvalk** [comment]
Thanks for asking! There's some ideas about this here https://github.com/pybricks/support/issues/401

In the mean time, perhaps you can get creative with using the output terminal as a way to show your data.
